cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/ClinicalTrialPhase.java

```diff
@@ -0,0 +1,58 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static java.util.Arrays.stream;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.stream.Collectors;
+import nva.commons.core.SingletonCollector;
+
+public enum ClinicalTrialPhase {
```

Comment:
Another thing. In our NVA model, do we want to have

`{
  ...
  "phase": "PHASE_ONE",
  ...
}`

or

`{
  ...
  "phase": "1",
  ...
}`

?

Reply:
I will change the enums in new PR to have to values instead of one. Then we can have one value for NVA and one value for Cristin

Reply:
Not sure I like the idea to mix all concerns in one enum. We need a simple and clear enum for our NVA model. From Cristin we can just use string if so. Mapping should be in a different package from the NVA model, totally decoupled from the NVA model.